src/search/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ endif()
 
 # On Windows, find the psapi library for determining peak memory.
 if(WIN32)
+    cmake_policy(SET CMP0074 NEW)
```

Comment:
Can you try what we discussed in http://issues.fast-downward.org/issue1003?

Reply:
I am not sure what you want me to try. First, we have to find a solution for the CMake requirements (discussion of issue1003). Even if we can increase the requirement only for Windows to >= 3.12 we still need to set the policy. The documentation for 3.19 tells us that if the policy is not explicitly set, it uses the old version

Reply:
You're right. Nothing to be done for that in this issue.